src/v2/view-builder/views/consent/EnduserEmailConsentView.js

```diff
@@ -40,8 +42,6 @@ const enduserEmailConsentViewBody = ConsentViewForm.extend({
 });
 
 export default BaseAuthenticatorView.extend({
-  className: 'enduser-email-consent',
```

Comment:
before
![Screen Shot 2022-01-04 at 4 20 48 PM](https://user-images.githubusercontent.com/71431120/148141431-aca77ce6-f5dd-4cff-a47d-4e3b412d705d.png)
after
![Screen Shot 2022-01-04 at 4 18 53 PM](https://user-images.githubusercontent.com/71431120/148141410-e28ef462-9bdc-4d28-910e-9ee233eec878.png)

Because the identifier styles are applied inside .siw-main-view, which overrides by setting classname here